src/Misc/Footer.js

```diff
@@ -0,0 +1,207 @@
+import React from "react"
+import styled from "styled-components"
+import { Grid, Typography, Paper, Link } from "@material-ui/core"
+import { breakpoints } from "../theme"
+import IconButton from "@material-ui/core/IconButton"
+import InstagramIcon from "@material-ui/icons/Instagram"
+import TwitterIcon from "@material-ui/icons/Twitter"
+import LinkedInIcon from "@material-ui/icons/LinkedIn"
+import FacebookIcon from "@material-ui/icons/Facebook"
+
+const Footer = () => {
+  return (
+    <Container>
+      <FooterSection alignItems="stretch">
+        <Paper
+          style={{
+            backgroundColor: "#fafafa",
+            borderRadius: "25px 25px 0 0",
+          }}
+          elevation={4}
+        >
+          <FooterGrid>
+            <Item>
+              <Section
+                xs={12}
+                top={30}
+                spacing={6}
+                item
+                container
+                justify="center"
+              >
+                <FooterColumn header="Get in touch">
+                  <P>Contact</P>
+                  <P>info@apfn.com</P>
+                  <P>+31 040 123 4567</P>
+                </FooterColumn>
+                <FooterColumn header="Legal">
+                  <FooterLink href="#">
+                    {/*links in here*/}
+                    <P>Terms and conditions</P>
+                  </FooterLink>
+                  <FooterLink href="#">
+                    {/*links in here*/}
+                    <P>Privacy policy</P>
+                  </FooterLink>
+                </FooterColumn>
+                <FooterColumn header="Learn more">
+                  <FooterLink href="#">
+                    {/*links in here*/}
+                    <P>How it works</P>
+                  </FooterLink>
+                  <FooterLink href="#">
+                    {/*links in here*/}
+                    <P>About Us</P>
+                  </FooterLink>
+                  <FooterLink href="#">
+                    {/*links in here*/}
+                    <P>FAQ</P>
+                  </FooterLink>
```

Comment:
You could decrease code size by alot by using Array.map(item => <Item .... />).
Check https://reactjs.org/docs/lists-and-keys.html

Reply:
I did this for the social media links part but it lead to a much shorter code and calling Material UI components like <FacebookIcon/> from a list didn't work as I thought it would. I think it is fine/it should stay like this for now. Trying to implement this with lists was a good learning though, so thanks.